todo-api/index.js

```diff
@@ -0,0 +1,33 @@
+var express = require("express");
+var cors = require("cors");
+var axios = require("axios");
+var bodyParser = require("body-parser");
+var app = express();
+app.use(cors());
+app.use(bodyParser.json());
+
+app.get("/", function (req, res) {
+  res.end("Server running at http://localhost:8081/");
+});
+
+const apiUrl = "https://illfatedlavendermemoryallocator--jabez007.repl.co/api";
```

Comment:
Would it have been possible to create an Axios instance here for simplicity and added flexibility in future use cases?